docs/src/pages/keymap-upgrader.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Keymap Upgrader
+sidebar_label: Keymap Upgrader
+hide_title: true
+hide_table_of_contents: true
+---
+
+# Keymap Upgrader
+
+Some behaviors, key codes, and other features have been renamed to be more consistent with each other. This tool will upgrade most deprecated features to their replacements.
+
+Paste the contents of a `.keymap` file below. Then, hover your mouse over the upgraded keymap and click the `Copy` button in the upper-right corner to copy it to your clipboard.
```

Comment:
```suggestion
Paste the contents of a `.keymap` file below. Then, hover your mouse over the upgraded keymap and click the `Copy` button in the upper-right corner to copy it to your clipboard.
Lines that have modified content will be highlighted.
```
Not mentioning the line deletion scenario but I think we can keep it simple.

Reply:
Done.